terraform/monitoring/02_dashboards.tf

```diff
@@ -0,0 +1,117 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Here we create a dashboard for the general user experience - it contains metrics
+# that reflect how users are interacting with the demo application such as latency
+# of requests, distribution of types of requests, and response types.
+resource "google_monitoring_dashboard" "userexp_dashboard" {
+	dashboard_json = <<EOF
+	{
+  "displayName": "User Experience Dashboard",
+  "gridLayout": {
+    "columns": "2",
+    "widgets": [
+      {
+        "title": "HTTP Request Count by Method",
+        "xyChart": {
+          "dataSets": [
+            {
+              "timeSeriesQuery": {
+                "timeSeriesFilter": {
+                  "filter": "metric.type=\"custom.googleapis.com/opencensus/opencensus.io/http/server/request_count_by_method\"",
```

Comment:
FYi @gzhao408, if Nic's PR is merged before your work on OC->OT, we will need to update the name of the metric after we move to OT

Reply:
There are a few more places like that in terraform monitoring

Reply:
Got it, will remember to replace these lines when I change the metrics